apps/react-18-tests-v8/package.json

```diff
@@ -10,21 +10,19 @@
     "e2e": "cypress run --component",
     "e2e:local": "cypress open --component",
     "lint": "just-scripts lint",
-    "test": "just-scripts test",
-    "just": "just-scripts",
+    "test": "jest --passWithNoTests",
     "start": "webpack-dev-server --mode=development --config=webpack.config.js",
     "type-check": "tsc -b tsconfig.json"
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "*",
-    "@fluentui/scripts": "^1.0.0",
-    "@types/react": "18.0.14",
-    "@types/react-dom": "18.0.6",
-    "swc-loader": "^0.2.3"
```

Comment:
swc-loader is in root package version already - single version policy conformance